src/cli.rs

```diff
@@ -79,6 +82,9 @@ pub struct ChangelogCommand {
     /// Print hidden sections
     #[clap(long)]
     pub include_hidden_sections: bool,
+    /// Only commits that update those files/folders will be taken into account. It is useful to support monorepos.
+    #[clap(short = 'F', long = "file")]
```

Comment:
same remark here as for the version command

```suggestion
    #[clap(short, long)]
```